deepinv/datasets/cbsd68.py

```diff
@@ -0,0 +1,93 @@
+from typing import Any, Callable
+import os
+
+import datasets
+import torch
+
+from deepinv.datasets.utils import calculate_md5
+
+
+class CBSD68(torch.utils.data.Dataset):
+    """Dataset for `CBSBD68 <https://paperswithcode.com/dataset/cbsd68>`_.
+
+    Color BSD68 dataset for image denoising benchmarks is part of The Berkeley Segmentation Dataset and Benchmark.
+    It is used for measuring image denoising algorithms performance. It contains 68 images.
+
+
+    **Raw data file structure:** ::
+
+            self.root --- data-00000-of-00001.arrow
+                       -- dataset_info.json
+                       -- state.json
+
+    This dataset wraps the huggingface version of the dataset.
+    HF source : https://huggingface.co/datasets/deepinv/CBSD68
+
+    :param str root: Root directory of dataset. Directory path from where we load and save the dataset.
+    :param bool download: If ``True``, downloads the dataset from the internet and puts it in root directory.
+        If dataset is already downloaded, it is not downloaded again. Default at False.
+    :param callable, optional transform: A function/transform that takes in a PIL image
+        and returns a transformed version. E.g, ``torchvision.transforms.RandomCrop``
```

Comment:
can you add a (lightweight) doctest example?

Reply:
I am unsure of what is a "doctest" example